### DISCUSSION TITLE
woodpecker agent docker image latest tag broken

### DISCUSSION THREAD

**Discussion by pixielark** [woodpecker-ci/woodpecker#4323] [Ask a Question] [answered]
it seems like after upgrade woodpecker server to v2.7.3, woodpecker agent docker stop working with error message saying grpc version mismatch, agent using version 9 but server using 11

I saw woodpecker agent docker image with latest tag update 2 hours ago on docker hub but still showing the same error message

however switch to next tag the agent seems to be working just fine

so are we expecting to use next tag instead of latest tag for woodpecker agent now?

Thanks

**Comment by qwerty287** [comment, marked as answer]
Agent and server must use the same version.

**Comment by zc-devs** [comment]
Maybe you hit https://github.com/woodpecker-ci/woodpecker/pull/4303#issuecomment-2458435211

**Comment by pixielark** [comment]
i will for now stick with `next` release for both my service and agent since I updated agent to `next` and wasn't aware that its also changing DB
